paddle/fluid/framework/details/op_handle_base.h

```diff
@@ -31,7 +31,9 @@ constexpr char kLocalExecScopeName[] = "@LCOAL_SCOPE@";
 // It's responsible for populating necessary fields of ir::Node.
 class OpHandleBase {
  public:
-  explicit OpHandleBase(ir::Node *node) : node_(node) {}
+  explicit OpHandleBase(ir::Node *node) : node_(node) {
```

Comment:
After a bit discuss, it seems it is better that the Creation function can be ` OpHandleBase(ir::Node *node, bool wrapped_by_node=False)`.